webapp/src/main/webapp/index.html

```diff
@@ -9,15 +9,6 @@
   <body>
     <div id="content">
       <h1><a href="index.html">GTFS Validator</a></h1>
-      <h2>About</h2>
-      <p>The General Transit Feed Specification (GTFS) is an open data standard. It enables transit agencies to share their data describing their transit service in a standard format that can be used by rider-facing software applications. </p>
-      <h2>How to use</h2>
-      <p>Upload a compressed .zip file including all of the individual files for your transit service.</p>
-      <form method="post" action="fileupload" enctype="multipart/form-data">
-        <input type="file" name="file" id="file" accept=".zip" data-multiple-caption="{count} files selected"/>
-        <br>
-        <button class="btn btn-outline-primary" type="submit">Upload</button>
-      </form>
     </div>
```

Comment:
Why is this file changing?

Reply:
I removed it from here to include it in another PR request.

Reply:
Please be careful when merging the PRs.